backend/src/test/java/com/votogether/domain/category/service/CategoryServiceTest.java

```diff
@@ -0,0 +1,170 @@
+                .name("개발")
+                .build();
+
+        categoryRepository.save(category);
+
+        // when
+        List<CategoryResponse> categories = categoryService.getAllCategories();
+
+        // then
+        assertAll(
+                () -> assertThat(categories.get(0).id()).isNotNull(),
+                () -> assertThat(categories.get(0).name()).isEqualTo("개발"),
+                () -> assertThat(categories.get(0).isFavorite()).isFalse()
+        );
+    }
+
+    @Test
+    @DisplayName("선호하는 카테고리를 선호 카테고리 목록에 추가한다.")
+    void addFavoriteCategory() {
+        // given
+        Category category = Category.builder()
+                .name("개발")
+                .build();
+
+        Member member = Member.builder()
+                .gender(Gender.MALE)
+                .point(0)
+                .socialType(SocialType.GOOGLE)
+                .nickname("user1")
+                .socialId("kakao@gmail.com")
+                .birthDate(
+                        LocalDateTime.of(1995, 7, 12, 0, 0))
+                .build();
+
+        categoryRepository.save(category);
+        memberRepository.save(member);
+
+        Long categoryId = category.getId();
+
+        // when
+        categoryService.addFavoriteCategory(member, categoryId);
+
+        // then
+        MemberCategory memberCategory = memberCategoryRepository.findByMemberAndCategory(member, category).get();
+
+        Assertions.assertAll(
+                () -> assertThat(memberCategory.getMember()).isSameAs(member),
+                () -> assertThat(memberCategory.getCategory()).isSameAs(category)
+        );
+    }
+
+    @Nested
+    @DisplayName("카테고리 삭제")
+    class Deleting {
+
+        @Test
+        @DisplayName("선호하는 카테고리를 선호 카테고리 목록에 삭제한다.")
+        void removeFavoriteCategory() {
+            // given
+            Category category = Category.builder()
+                    .name("개발")
+                    .build();
+
+            Member member = Member.builder()
+                    .gender(Gender.MALE)
+                    .point(0)
+                    .socialType(SocialType.GOOGLE)
+                    .nickname("user1")
+                    .socialId("kakao@gmail.com")
+                    .birthDate(
+                            LocalDateTime.of(1995, 7, 12, 0, 0))
+                    .build();
+
+            MemberCategory memberCategory = MemberCategory.builder()
+                    .member(member)
+                    .category(category)
+                    .build();
+
+            categoryRepository.save(category);
+            memberRepository.save(member);
+            memberCategoryRepository.save(memberCategory);
+
+            Long categoryId = category.getId();
+
+            // when
+            categoryService.removeFavoriteCategory(member, categoryId);
+
+            // then
+            Optional<MemberCategory> foundMemberCategory = memberCategoryRepository.findByMemberAndCategory(member,
+                    category);
```

Comment:
```suggestion
Optional<MemberCategory> foundMemberCategory =
                memberCategoryRepository.findByMemberAndCategory(member,category);
```
`P2` : IDE가 아니라서 조금 어색하긴한데 이런 느낌의 개행이 더 가독성이 좋은 것 같은데 어떻게 생각하시나요?